core/src/test/java/nl/nn/adapterframework/http/rest/OpenApiTest.java

```diff
@@ -21,8 +22,10 @@
 	public void simpleEndpointTest() throws Exception {
 		ApiServiceDispatcher dispatcher = ApiServiceDispatcher.getInstance();
 		assertEquals("there are still registered patterns! Threading issue?", 0, dispatcher.getPatternClients().size());
-
-		new AdapterBuilder("myAdapterName", "description4simple-get").setListener("users", "get").setValidator("simple.xsd", null, "user").build(true);
+		Parameter param = new Parameter();
+		param.setName("parameter");
+		param.setValue("parameter");
+		new AdapterBuilder("myAdapterName", "description4simple-get").setListener("users", "get", "operationId4simple-get").setValidator("simple.xsd", null, "user", param).build(true);
 
```

Comment:
Please create a new test for every new thing you add.. Right now this single method is testing over 10 different things..